image_pipeline/doc/index.rst

```diff
@@ -0,0 +1,43 @@
+Overview
+========
+
+The ``image_pipeline`` stack is designed to process raw camera images
+into useful inputs to vision algorithms: rectified mono/color images,
+stereo disparity images, and stereo point clouds. Components include:
+
+ * **Calibration**: Cameras must be calibrated in order to relate the
+   images they produce to the three-dimensional world. The
+   ``camera_calibration`` package provides tools to calibrate monocular
+   and stereo cameras in your ROS system. The :ref:`Camera Info` page
+   provides a detailed description of the parameters used by the
+   pipeline.
+ * **Monocular processing**: The raw image stream can be piped through
+   the ``image_proc`` node to remove camera distortion. The node also
+   performs color interpolation for Bayer pattern color cameras.
+ * **Stereo processing**: The ``stereo_image_proc`` package performs
+   the duties of ``image_proc`` for a pair of cameras co-calibrated
+   for stereo vision. It also uses stereo processing to produce
+   disparity images and point clouds.
+ * **Depth processing**: ``depth_image_proc`` provides components
+   for processing depth images (as produced by the Kinect,
+   time-of-flight cameras, etc.), such as producing point clouds.
+ * **Visualization**: The ``image_view`` package provides a lightweight
+   alternative to ``rviz2`` for viewing an image topic. It also includes
+   a ``stereo_view`` tool for viewing stereo pairs and disparity images.
+
+.. toctree::
+   :maxdepth: 2
+
+   self
+   camera_info
+   camera_calibration <../camera_calibration/index.html#http://>
```

Comment:
this is a little hacky, but according to stack overflow - it's literally the only way to do a relative link in a TOC - if you don't have http:// in there somewhere, it tries to treat it as a filename, which obviously doesn't exist at build time.